packages/browser/src/browser/index.ts

```diff
@@ -363,7 +363,7 @@ async function loadAnalytics(
   if (!options.disableClientPersistence && options.httpCookieServiceOptions) {
     options.httpCookieService = await HTTPCookieService.load(
       options.httpCookieServiceOptions
-    )
+    ).catch((err): undefined => console.error(err) as undefined)
```

Comment:
We dont want HTTPCookieService to blow up regular tracking, so catch errors here.